components/widgets/CurrentData/patterns/Daylight/index.js

```diff
@@ -0,0 +1,219 @@
+                  </tspan>
+                );
+              }
+
+              if (time === 12) {
+                const noon = date.toLocaleTimeString(language, {
+                  dayPeriod: "short",
+                  timeZone: timezone,
+                });
+
+                return (
+                  <tspan
+                    style={{
+                      fontSize: variant === "primary" && "10px",
+                    }}
+                  >
+                    {capitalize(noon, language)}
+                  </tspan>
+                );
+              }
+            },
+          }}
+        />
+        <g role="list">
+          <Styled.BoundRect
+            y={0}
+            x={xScale(0)}
+            width={xScale(dawnMinutes)}
+            height={xPos}
+            role="listitem"
+            aria-label={t(
+              "summit_dashboard.sections.astro.daylight.observing",
+              {
+                from: formatTime(midnight, language),
+                to: formatTime(dawn, language),
+              }
+            )}
+          />
+          <Styled.BoundRect
+            style={{ "--fill": twilightFill }}
+            y={0}
+            x={xScale(dawnMinutes)}
+            width={xScale(sunriseMinutes - dawnMinutes)}
+            height={xPos}
+            role="listitem"
+            aria-label={t("summit_dashboard.sections.astro.daylight.twilight", {
+              from: formatTime(dawn, language),
+              to: formatTime(sunrise, language),
+            })}
+          />
+          <Styled.BoundRect
+            style={{ "--fill": dayFill }}
+            y={0}
+            x={xScale(sunriseMinutes)}
+            width={xScale(sunsetMinutes - sunriseMinutes)}
+            height={xPos}
+            role="listitem"
+            aria-label={t("summit_dashboard.sections.astro.daylight.daylight", {
+              from: formatTime(sunrise, language),
+              to: formatTime(sunset, language),
+            })}
+          />
+          <Styled.BoundRect
+            style={{ "--fill": twilightFill }}
+            y={0}
+            x={xScale(sunsetMinutes)}
+            width={xScale(nightMinutes - sunsetMinutes)}
+            height={xPos}
+            role="listitem"
+            aria-label={t("summit_dashboard.sections.astro.daylight.twilight", {
+              from: formatTime(sunset, language),
+              to: formatTime(night, language),
+            })}
+          />
+          <Styled.BoundRect
+            y={0}
+            x={xScale(nightMinutes)}
+            width={xScale(minutes - nightMinutes)}
+            height={xPos}
+            role="listitem"
+            aria-label={t(
+              "summit_dashboard.sections.astro.daylight.observing",
+              {
+                from: formatTime(night, language),
+                to: formatTime(midnight, language),
+              }
+            )}
+          />
+        </g>
+        <g>
+          <Styled.Now
```

Comment:
the "now" indicator will read out the current time and if it is dark enough to observe